lib/linguist/heuristics.rb

```diff
@@ -323,7 +323,7 @@ def call(data)
     disambiguate ".ms" do |data|
       if /^[.'][a-z][a-z](\s|$)/i.match(data)
         Language["Roff"]
-      elsif /(?<!\S)\.(include|globa?l)\s/.match(data) || /(?<!\/\*)(\A|\n)\s*\.[A-Za-z][_A-Za-z0-9]*:/.match(data.gsub(/"([^\\"]|\\.)*"|'([^\\']|\\.)*'|\\\s*(?:--.*)?\n/, ""))
+      elsif !/\/\*/.match(data) && /^\s*\.(include\s|globa?l\s|[A-Za-z][_A-Za-z0-9]*:)/.match(data)
```

Comment:
It wouldn't hurt to make this a non-capturing group:

~~~
        ↓↓
/^\s*\.(?:include\s|globa?l\s|[A-Za-z][_A-Za-z0-9]*:)
        ↑↑
~~~